packages/radio/src/react/button.tsx

```diff
@@ -25,40 +23,48 @@ const styles = {
   circleInner: () => css(stylesheet['.psds-radio-button__circle-inner']),
   halo: () => css(stylesheet['.psds-radio-button__halo']),
   input: () => css(stylesheet['.psds-radio-button__input']),
-  label: themeName =>
+  label: (themeName: ValueOf<typeof names>) =>
     compose(
       css(stylesheet['.psds-radio-button__label']),
       css(stylesheet[`.psds-radio-button__label.psds-theme--${themeName}`])
     )
 }
 
-const isChecked = (a, b) => a === b
+const isChecked = (a: React.ReactText, b?: React.ReactText) => a === b
 
-const Button = React.forwardRef(({ value, label, ...props }, forwardedRef) => {
+export interface RadioButtonProps extends Omit<HTMLAttributes<HTMLInputElement>, 'onClick'> {
+  label: React.ReactNode
+  onBlur?: (e?: React.FocusEvent) => void
+  onClick?: (e?: React.MouseEvent, v?: React.ReactText) => void
```

Comment:
I'd give more than one letter names to these params.  `e` is perhaps easily guessable. I'd go for `evt` or `event` to make sure.  `v`, I really don't know without reading more code.